api/src/pcapi/routes/backoffice/venues/blueprint.py

```diff
@@ -560,6 +570,28 @@ def delete_venue(venue_id: int) -> utils.BackofficeResponse:
     return redirect(url_for("backoffice_web.pro.search_pro"), code=303)
 
 
+@venue_blueprint.route("/<int:venue_id>/fully-sync", methods=["POST"])
+@utils.permission_required(perm_models.Permissions.MANAGE_PRO_ENTITY)
+def fully_sync_venue(venue_id: int) -> utils.BackofficeResponse:
+    venue = (
+        offerers_models.Venue.query.join(providers_models.VenueProvider)
+        .join(providers_models.Provider)
+        .filter(providers_models.Provider.allow_bo_sync == True)
```

Comment:
Petit détail de syntaxe :  
```suggestion
        .filter(providers_models.Provider.allow_bo_sync.is_(True))
```